components/card/metadata/card.yml

```diff
@@ -8,10 +8,11 @@ sections:
 
       ### Change workflow icon size to medium
       Please replace `.spectrum-Icon--sizeS` with `.spectrum-Icon--sizeM`.
-      ### Use Spectrum Heading for title and Spectrum Detail for subtitle
-      To make the card component more flexible, products can have more fine control over the typography by using the Heading and Detail components with their corresponding sizes. To be more consistent with previous card designs you can use the following:
+      ### Use Spectrum Heading for title 
+      To make the card component more flexible, products can have more fine control over the typography by using the Heading component with its corresponding sizes. To be more consistent with previous card designs you can use the following:
         * Add the `spectrum-Heading` and `spectrum-Heading--sizeXS` or `spectrum-Heading--sizeXXS` to the `spectrum-Card-title` element.
```

Comment:
I wonder if it would be valuable to extend the typography styles in the card so that users don't have to add the classes and pull in the typography styles separately since they're quite detailed. Or is the flexibility of using any class desirable?

Reply:
I assumed we were wanting the flexibility of being able to use any class but that's not a call for me to make!